operations/observability/mixins/Makefile

```diff
@@ -1,44 +1,48 @@
+.DEFAULT_GOAL:=help
+
+# HELP
+# https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+.PHONY: help
+
+help:  ## This help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
 JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
 
-all: fmt lint
+all: fmt lint ## Format and lint jsonnet files
 
-.PHONY: clean
-clean:
-    # Delete files marked in .gitignore
+clean: ## Delete files marked in .gitignore
 	git clean -Xfd .
 
-.PHONY: fmt
-fmt:
+fmt: ## Format jsonnet files
 	find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
 		xargs -n 1 -- $(JSONNET_FMT) -i
 
-.PHONY: lint
-lint:
+lint: ## Lint jsonnet files
 	find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
 		while read f; do \
 			$(JSONNET_FMT) "$$f" | diff -u "$$f" -; \
 		done
 
-.PHONY: prometheus_alerts.yaml
-prometheus_alerts.yaml:
+prometheus_alerts.yaml: ## Create prometheus alerts yaml file
 	jsonnet -S alerts.jsonnet > $@
 
-.PHONY: promtool-lint
-promtool-lint: prometheus_alerts.yaml
+promtool-lint: prometheus_alerts.yaml # Check generated prometheus alerts yaml file
 	promtool check rules prometheus_alerts.yaml
 
-.PHONY: vendor
 vendor:
 	jb install
+	# Custom version of https://github.com/grafana/grafonnet-lib/pull/274
+	cp components/bar_gauge_panel.libsonnet vendor/github.com/grafana/grafonnet-lib/grafonnet/bar_gauge_panel.libsonnet
```

Comment:
This works when developing locally, but won't work when importing those changes to our observability repository

We have another vendor file over there and it we can't move directories around when rendering dashboards with ArgoCD 😕